apps/store/src/components/PriceCalculator/CurrentInsuranceField/CurrentInsuranceField.tsx

```diff
@@ -14,7 +13,7 @@ type Props = {
 }
 
 export const CurrentInsuranceField = ({ label }: Props) => {
-  const priceIntent = useAtomValue(priceIntentAtom)
+  const priceIntent = usePriceIntent()
```

Comment:
nit: It would be nice to highlight in the name for hooks that returns an atom value 
```suggestion
  const priceIntent = usePriceIntentAtom()
```